### PR TITLE
feat(blog): add Camunda Modeler v3.3.0 release blog post

### PR DESCRIPTION
**Pull request by philippfromme**
Looking forward to your feedback. ❤️ 

Related to camunda/camunda-modeler#1480

### CODE REVIEW

File: content/post/2019/08/camunda-modeler-3.3.0-released.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+{{< figure class="no-border teaser" src="navigation.gif" alt="Navigation" caption="No accidentally moving elements instead of the canvas anymore" >}}
+
+Furthermore, you can now press `Space` to activate the hand tool momentarily. Alternatively, press `CommandOrControl` when dragging to move the canvas.
+
+### Better Copy & Paste Experience
+
+Copying and pasting elements may sound like a straight forward feature. Nevertheless, we thought we could do better. Why wouldn't you want all the great features you already get when creating new elements? Features like snapping to existing elements, snapping to the grid and seeing what your elements will look like once you've created them. As of this release pasting elements works just like creating them because that is what you're doing.
+
+{{< figure class="no-border teaser" src="copy-paste.gif" alt="Copy Paste" caption="Copy Paste " >}}
```
Comment (barmac) [inline content/post/2019/08/camunda-modeler-3.3.0-released.md]: Let's have some meaningful caption here, shall we? E.g.:

_Copy and paste in two steps for greater precision._

**Comment by barmac** [comment]
Overall, it looks good to me. I left one comment and IMO with that slight change it can be merged.

**Comment by nikku** [comment]
Should we mention that copy and paste has been completely reworked (not only interaction, but internals, too).

From our point of view it is now completely fixed.

**Comment by nikku** [comment]
> We are proud to announce the release of Camunda Modeler v3.3.0. It comes with a couple of significant improvements to the BPMN modeling experience as well as many bug fixes.

Are we that vague all the time? If I read a blog post I usually like to get teasered (i.e. new copy and paste interaction / navigation / everything).

**Comment by philippfromme** [comment]
> Should we mention that copy and paste has been completely reworked (not only interaction, but internals, too).
> 
> From our point of view it is now completely fixed.

Well, do we want people to know about how broken it was before?

**Comment by nikku** [comment]
Not looking back, but looking into the future, yes. Those who know about the current limitations may see that as a major advantage.

**Comment by nikku** [comment]
Regarding the past => "fix existing limitations" sounds much better than everything was lost :see_no_evil:

**Comment by nikku** [comment]
Your choice :wink:

**Comment by philippfromme** [comment]
"Ricarding the limitations of copy & paste, it's all good now."

File: content/post/2019/08/camunda-modeler-3.3.0-released.md (hunk 1 above)
Comment (philippfromme) [reply content/post/2019/08/camunda-modeler-3.3.0-released.md]: Sorry, this is actually a mistake. I forgot about that.

Comment (barmac) [reply content/post/2019/08/camunda-modeler-3.3.0-released.md]: Kein Ding.

**Comment by philippfromme** [comment]
Thank you for your feedback. I've made a few adjustments. Let's merge. 🔥

File: content/post/2019/08/camunda-modeler-3.3.0-released.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+title = "Camunda Modeler 3.3.0 Released"
++++
+
+We are proud to announce the release of Camunda Modeler v3.3.0. It comes with two significant improvements to our BPMN modeler: improved navigation and a better copy & paste experience. Furthermore, we included several bug fixes.
+
+[Download the latest release](https://camunda.com/download/modeler/) and start modeling.
+
+<!--more-->
+
+### Improved Navigation
```
Comment (nikku) [inline content/post/2019/08/camunda-modeler-3.3.0-released.md]: Is it intentional that we use two different kinds of heading here?

Comment (philippfromme) [reply content/post/2019/08/camunda-modeler-3.3.0-released.md]: Fixed.

**Comment by philippfromme** [comment]
Have we decided whether the title will be _Camunda Modeler v3.3.0_ or _v3.3_? @nikku

**Comment by barmac** [comment]
Yes, we decided for 3.3

**Comment by barmac** [comment]
Updated the title and the release date (today, 4 PM).

**Pull request merged**